spec/dummy_controller_spec.rb

```diff
@@ -373,5 +373,25 @@
         expect(response_body(response)['items']).to eq expected_list
       end
     end
+
+    context 'when paginating an ActiveRecord with a group by query' do
+      let!(:dummy_models) { create_list(:dummy_model, 12, name: 'argentina') }
```

Comment:
We can do this without `!` and only creating 1 dummy_model by group
```rb
let(:argentina) { create(:dummy_model, name: 'Argentina') }
let(:uruguay) { create(:dummy_model, name: 'Uruguay') }
let(:costa_rica) { create(:dummy_model, name: 'Costa Rica') }
```

Reply:
I set count now to 1, 2 and 3 because with 1 instance per group we doesn't proove  the group by functionality